src/main/java/teammates/logic/api/TaskQueuer.java

```diff
@@ -294,6 +294,21 @@ public void scheduleEmailsForSending(List<EmailWrapper> emails) {
         }
     }
     
+    /**
+     * Schedules production of searchable documents
```

Comment:
Option 1: Fixed waits are problematic given the vast difference in performance between dev server and a remote server. We should figure out a reasonable way to do dynamic waits (with a timeout). Don't we have something similar already? 
Option 2: Bay be its time to take a shot at #2258 ?